packages/plugin-core/src/showcase/GraphThemeTip.ts

```diff
@@ -23,7 +23,7 @@ export class GraphThemeTip implements IFeatureShowcaseMessage {
   getDisplayMessage(displayLocation: DisplayLocation): string {
```

Comment:
since we are doing this more often, can we create a command that is enabled in development called "Show Tip" which chooses what tip is currently shown? will help with debugging. can be a next item